Gemfile

```diff
@@ -29,6 +29,7 @@ gem 'uglifier', '>= 1.3.0'
 group :development, :test do
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
   gem 'factory_bot_rails', '~> 4.0'
+  gem 'shoulda-matchers', '~> 3.1'
```

Comment:
why moved here, not in test group ?

Reply:
because in dev environment, rspec don't work without shoulda_matchers

Reply:
in moved here for have shoulda_matchers in all environment test and dev